tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -473,21 +553,49 @@ def test_check_sfputil_reset(duthosts, enum_rand_one_per_hwsku_frontend_hostname
             if duthost.facts["asic_type"] == "vs" and sfp_presence['rc'] == 2:
                 pass
             else:
-                assert sfp_presence['rc'] == 0, \
-                    "Run command '{}' failed".format(cmd_sfp_presence_per_intf)
+                assert sfp_presence['rc'] == 0, (
+                    "Run command '{}' failed with return code {}. "
+                    "This means the 'sfputil show presence' command did not execute successfully "
+                    "on the DUT for the specified interface. "
+                    "Please check the command output, DUT logs, and platform status for troubleshooting."
+                ).format(cmd_sfp_presence_per_intf, sfp_presence.get('rc', 'N/A'))
 
             parsed_presence = parse_output(sfp_presence["stdout_lines"][2:])
-            assert logical_intf in parsed_presence, \
-                "Interface is not in output of '{}'".format(cmd_sfp_presence_per_intf)
-            assert parsed_presence[logical_intf] == "Present", \
-                "Interface presence is not 'Present' for {}".format(logical_intf)
+            assert logical_intf in parsed_presence, (
+                "Interface '{}' is not in output of '{}'. "
+                "This means the SFP presence information for the interface is missing after reset. "
+                "- Parsed Presence Output: {}\n"
+            ).format(
+                logical_intf,
+                cmd_sfp_presence_per_intf,
+                parsed_presence
+            )
+            assert parsed_presence[logical_intf] == "Present", (
+                "Interface presence is not 'Present' for '{}'. Got: '{}'. "
+                "This means the SFP module is not detected as present after reset."
+            ).format(
+               logical_intf,
+               parsed_presence[logical_intf]
+            )
 
     # Check interface status for all interfaces in the end just in case
-    assert check_interface_status(duthost,
-                                  [logical_intf
-                                   for logical_intfs_dict in phy_intfs_to_test_per_asic.values()
-                                   for logical_intf, is_admin_up in logical_intfs_dict.items() if is_admin_up],
-                                  expect_up=True)
+    assert check_interface_status(
+        duthost,
+        [logical_intf
+         for logical_intfs_dict in phy_intfs_to_test_per_asic.values()
+         for logical_intf, is_admin_up in logical_intfs_dict.items() if is_admin_up],
+        expect_up=True
+    ), (
+        "Not all interfaces that are admin up transitioned to the 'up' state after SFP reset. "
+        "This means one or more interfaces expected to be operationally up are still down. "
```

Comment:
"This means.." is not necessary at all. Just tell the failed reason, no need to tell user how to debug or fix it. Make the summary clean and clear.